pandas/core/frame.py

```diff
@@ -2889,11 +2889,11 @@ def _set_value(self, index, col, value, takeable=False):
 
     _set_value.__doc__ = set_value.__doc__
 
-    def _ixs(self, i, axis=0):
+    def _ixs(self, i: int, axis: int = 0):
```

Comment:
Since `axis` is a pretty common keyword can you put that as `Axis` in pandas._typing? We probably also want to add `str` as a type for it since it *should* accept "index" and "columns" as well

Reply:
Also can we add the return type here? `-> 'DataFrame`'?

Reply:
I'll take a look and see what we can determine about return types.  For Axis/str/int, in all these cases, I've put int because these are private methods and only ever get ints.